dance/datasets/multimodality.py

```diff
@@ -317,7 +429,12 @@ def _maybe_preprocess(self, raw_data, selection_threshold=10000):
                     m2_test = lsi_transformer_atac.transform(modalities[3]).values
 
                 else:
-                    raise ValueError(f"Unrecognized subtask name: {self.subtask}")
```

Comment:
You did not make sure the second modality is atac in else branch? Please update the codes here.

Reply:
I'm modifying this and testing some datasets

Reply:
Some data sets are relatively large, and other codes and some parameters need to be adjusted together, so the test time is a bit long.

Reply:
For now, it looks OK. Later we can try to unify the subtask name. I'll mark this issue as resolved

Reply:
OK,After testing, I just added some subtask names